data/categories.txt

```diff
@@ -3718,7 +3719,7 @@ pl:3^Toaleta
 pt:^WC
 pt-BR:^Banheiro
 ro:^Toaletă
-es:^Baños
+es:^WC
```

Comment:
Note: it means the primary translation for toilets will now be "WC". Is this really the most used term?

Reply:
It is very common